src/libcrun/container.h

```diff
@@ -85,6 +92,27 @@ typedef struct libcrun_container_status_s libcrun_container_status_t;
 typedef struct libcrun_container_s libcrun_container_t;
 typedef struct libcrun_context_s libcrun_context_t;
 
+struct container_entrypoint_s
+{
+  libcrun_container_t *container;
+  libcrun_context_t *context;
+  int has_terminal_socket_pair;
+  int terminal_socketpair[2];
+
+  /* Used by log_write_to_sync_socket.  */
+  int sync_socket;
+
+  int seccomp_fd;
+  int seccomp_receiver_fd;
+  int console_socket_fd;
+
+  int hooks_out_fd;
+  int hooks_err_fd;
+
+  struct custom_handler_s *custom_handler;
+  void *handler_cookie;
+};
```

Comment:
could this just be `struct container_entrypoint_s;` and leave the declaration where it was?

Reply:
yes moved back.